pyrate/core/shared.py

```diff
@@ -717,6 +718,14 @@ def _is_coherence(hdr):
            (hdr[ifc.INPUT_TYPE] == InputTypes.COH if ifc.INPUT_TYPE in hdr else False)
 
 
+def _is_baseline(hdr):
+    """
+    Convenience function to determine if file is interferogram
```

Comment:
*file is baseline

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee